### PR TITLE
Closes #340 | Implement Dataloader for emotes_3k

### LINKED ISSUE #340: Create dataset loader for EMoTES-3K
Dataloader name: `emotes_3k/emotes_3k.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?emotes_3k

|     Dataset| emotes_3k |
|-------------|---|
| Description | This dataset features examples of commonsense morality in both English and Filipino, complete with reasoning justifications |
| Subsets | - |
| Languages | tgl |
| Tasks | Commonsense Reasoning |
| License | Apache license 2.0 (apache-2.0) |
| Homepage | https://huggingface.co/datasets/NLPinas/EMoTES-3K |
| HF URL | https://huggingface.co/datasets/NLPinas/EMoTES-3K |
| Paper URL | https://rootroo.com/downloads/nlp4dh_iwclul_proceedings.pdf |

### PR DESCRIPTION
**Pull request by luckysusanto**
Closes #340 

### Checkbox
- [X] Confirm that this PR is linked to the dataset issue.
- [X] Create the dataloader script `seacrowd/sea_datasets/emotes_3k/emotes_3k.py` (please use only lowercase and underscore for dataset naming).
- [X] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [X] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [X] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [X] Confirm dataloader script works with `datasets.load_dataset` function.
- [X] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/emotes_3k/emotes_3k.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/emotes_3k/emotes_3k.py [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    Tasks.MORALITY_CLASSIFICATION, # Roberta moral or immoral classification
+    Tasks.INSTRUCTION_TUNING # FLAN-T5 Training
+] 
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class Emotes3K(datasets.GeneratorBasedBuilder):
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/emotes_3k/emotes_3k.py:75]:
```suggestion
class Emotes3KDataset(datasets.GeneratorBasedBuilder):
```

File: seacrowd/sea_datasets/emotes_3k/emotes_3k.py [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    """
+    Emotes3K consists of one human annotated dataset for the purpose of morality classification and instruction tuning.
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="emotes_3k_source",
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/emotes_3k/emotes_3k.py:85]: Maybe we can use `_DATASETNAME` here?

File: seacrowd/sea_datasets/emotes_3k/emotes_3k.py [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                {
+                    'entry_id'          : datasets.Value("string"),
+                    'Filipino'          : datasets.Value("string"),
+                    'English'           : datasets.Value("string"),
+                    'Annotation'        : datasets.ClassLabel(names=["Moral", "Immoral"]),
+                    'Explanation'       : datasets.Value("string"),
+                    'Personality Traits': datasets.Value("string"),
+                    'Topic'             : datasets.Value("string"),
+                    'Topic Name'        : datasets.Value("string")
+                }
```
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/emotes_3k/emotes_3k.py:149]: Can you run the `make check_file=...` script again? I don't think this is how the formatter usually formats this dictionary.

**Review by ljvmiranda921 — COMMENTED** [review]
Just have a few formatting nits. Can you run `make check_file=...` again?

File: seacrowd/sea_datasets/emotes_3k/emotes_3k.py (hunk 1 above)
Comment (luckysusanto) [reply seacrowd/sea_datasets/emotes_3k/emotes_3k.py:75]: Implemented!

File: seacrowd/sea_datasets/emotes_3k/emotes_3k.py (hunk 2 above)
Comment (luckysusanto) [reply seacrowd/sea_datasets/emotes_3k/emotes_3k.py:85]: Yeah, that sounds oke.
Changed!

File: seacrowd/sea_datasets/emotes_3k/emotes_3k.py (hunk 3 above)
Comment (luckysusanto) [reply seacrowd/sea_datasets/emotes_3k/emotes_3k.py:149]: I have an issue with running make check_file sadly :(. 
How was it supposed to be formatted, if you don't mind me asking?

Comment (MJonibek) [reply seacrowd/sea_datasets/emotes_3k/emotes_3k.py:149]: @luckysusanto As in previous PRs, I can format this file and commit it before merging

**Review by MJonibek — APPROVED** [review]
checked, LGTM

**Review by ljvmiranda921 — APPROVED** [review]
LGTM!

**Pull request merged**